changelog/15099.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+ui: Fixes issue setting username field when creating a user for userpass auth method
```

Comment:
not sure you need a changelog if it's a bug that was created in the 1.11 release and is not being backported any farther back.

Reply:
Ya that's a good call. Since the bug exists in a different way on older versions but doesn't seem to impact creating a user I don't think this is worth backporting. I'll remove the changelog. Thanks!